wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/ProvisionCredentialScreen.kt

```diff
@@ -149,12 +148,26 @@ fun ProvisionDocumentScreen(
                     }
 
                     is EvidenceRequestMessage -> {
-                        EvidenceRequestMessageView(
-                            evidenceRequest,
-                            provisioningViewModel = provisioningViewModel,
-                            walletServerProvider = walletServerProvider,
-                            documentStore = documentStore
-                        )
+                        // show a standard message that does not have a [messageType] defined
+                        if (evidenceRequest.messageType==null) {
+                            EvidenceRequestMessageView(
+                                evidenceRequest = evidenceRequest,
+                                provisioningViewModel = provisioningViewModel,
+                                walletServerProvider = walletServerProvider,
+                                documentStore = documentStore
+                            )
+                        } else {
+                            // handle showing a non-standard message according to the classification
+                            // of the defined [messageType]
+                            if (evidenceRequest.messageType=="evidence_request_complete"){
```

Comment:
...if we do decide to keep messageType, this should be an enum, not a string.

Reply:
I think leaving it as a String provides these benefits:
- lower complexity for just 1 detailed message type "evidence_request_complete".
  - I'd be more inclined to use enums if we know in advance that there are "3 types of messages" to render.
- it's more flexible across platforms (portability).
- enables the use of `messageType` as a unique identifier, for example "confirmation_XYZ", and render a tailored UI for the "XYZ" confirmation message that may have an image/color/font/etc.. different from the generic confirmation message UIs. 

Happy to change to an enum if you feel strongly about it.